packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.gen.ts

```diff
@@ -58,6 +58,28 @@ export const defaultDashboardV2Spec = (): DashboardV2Spec => ({
 	schemaVersion: 39,
 });
 
+export interface LibraryPanelKind {
+	kind: "LibraryPanel";
+	spec: LibraryPanelSpec;
+}
+
+export const defaultLibraryPanelKind = (): LibraryPanelKind => ({
+	kind: "LibraryPanel",
+	spec: defaultLibraryPanelSpec(),
+});
+
+export interface LibraryPanelSpec {
+	// Library panel name
+	name: string;
+	// Library panel UID
+	uid: string;
+}
+
+export const defaultLibraryPanelSpec = (): LibraryPanelSpec => ({
+	name: "",
+	uid: "",
+});
```

Comment:
I'm not convinced we need to generate defaults for specs/kinds that are meaningless when not configured correctly. Like- who and why would use such a default  config?

Reply:
They are generated automatically when the field is required. I agree they are not needed or meaningful. Maybe we can add a non-default decorator or improve the default generation.